released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.0.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Cyril Cohen" "Kazuhiko Sakaguchi" "Enrico Tassi" ]
+license: "MIT"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+dev-repo: "git+https://github.com/math-comp/hierarchy-builder"
+
+build: [ make ]
```

Comment:
Is there a particular reason parallel make is not enabled? It looks like HB consists of many files, so it would likely help.

Reply:
It used to be broken, but now -j works.